extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/TenantContextFactory.java

```diff
@@ -223,6 +223,15 @@ private Uni<TenantConfigContext> createTenantContext(OidcTenantConfig oidcTenant
             throw new ConfigurationException(
                     "UserInfo is not required but 'verifyAccessTokenWithUserInfo' is enabled");
         }
+        if (!oidcConfig.authentication().idTokenRequired().orElse(true) && OidcUtils.isWebApp(oidcConfig)
```

Comment:
I'm sorry but I'm not sure this warning is better than failing the build, let's do just that - if it ever proves too restrictive we can revisit

Reply:
@michalvavrik Sure, let's leave it as a warning

Reply:
appreciate it, thank you